hazelcast/src/main/java/com/hazelcast/jet/pipeline/FileSourceBuilder.java

```diff
@@ -137,11 +137,7 @@ public FileSourceBuilder charset(@Nonnull Charset charset) {
         String charsetName = charset.name();
         return batchFromProcessor("filesSource(" + new File(directory, glob) + ')',
                 SourceProcessors.readFilesP(directory, glob, sharedFileSystem,
-                        path -> {
-                            String fileName = path.getFileName().toString();
-                            return Files.lines(path, Charset.forName(charsetName))
-                                        .map(l -> mapOutputFn.apply(fileName, l));
-                        }));
+                        SecuredFunctions.readFileFn(directory, charsetName, mapOutputFn)));
```

Comment:
Should we use a secured function also at these places?

https://github.com/hazelcast/hazelcast/pull/19249/files#diff-e87a79dc7bb0a897688b56a68c03c8f3fa48a3f0d757bff72dadbfc647f36605L171

https://github.com/hazelcast/hazelcast/pull/19249/files#diff-e87a79dc7bb0a897688b56a68c03c8f3fa48a3f0d757bff72dadbfc647f36605L237

Reply:
And here

https://github.com/hazelcast/hazelcast/blob/master/extensions/avro/src/main/java/com/hazelcast/jet/avro/AvroSourceBuilder.java#L105